charts/seed-bootstrap/templates/prometheus/statefulset.yaml

```diff
@@ -26,15 +26,16 @@ spec:
       - name: prometheus
         image: {{ index .Values.global.images "prometheus" }}
         imagePullPolicy: IfNotPresent
+        command: ["/bin/sh","-c"]
```

Comment:
is this reacting to `SIGTERM`?

Reply:
Yes, doing `kubectl delete pod prometheus-0` will terminate the pod without it hanging. PID 1 in the prometheus container is `/bin/prometheus <flags>`